Dockerfile

```
@@ -44,10 +44,19 @@ RUN apt-get update && \
 #ENV HTTP_PROXY=${http_proxy} \
 #    HTTPS_PROXY=${https_proxy} \
 #    NO_PROXY=${no_proxy}
-#COPY MY_ORG_ROOT.crt /usr/local/share/ca-certificates
-#RUN cd /usr/local/share/ca-certificates && \
-#    wget http://pki.my-organization.local/MY%20ORG%20ROOT.crt && \
-#    update-ca-certificates
+
+# Handle custom CA certificate, if specified.
+RUN if test -n "${TRAM_CA_URL}" -a -n "${TRAM_CA_THUMBPRINT}" ; then \
+        echo "Installing certificate authority from ${TRAM_CA_URL}" && \
```

```suggestion
        printf "Installing certificate authority from ${TRAM_CA_URL}" && \
```

Based on GitHub Action `/github/workspace/Dockerfile:49 SC2028 info: echo may not expand escape sequences. Use printf.`